src/class-savvii-dashboard.php

```diff
@@ -162,13 +155,82 @@ function warpdrive_dashboard() {
                     <!-- /Caching defaults -->
             <?php endif; ?>
                     <!-- Read server logs -->
-                    <div class="postbox" style="min-height: 150px;">
+                    <div class="postbox">
                         <h2 class="hndle">Read server logs</h3>
-                        <div class="inside"><?= is_super_admin() ? 'Please use the Savvii top menu for reading logs.' : '
-                            \'Read server logs\' shows data for all subsites. Because of this, only users with the \'Super Admin\' role are able to read the server logs.'; ?>
+                        <div class="inside">
+                            <?php
+                            if (is_super_admin()) {
+                            ?>
+                                <div class='row'>  
+                                    <div class='column'>
+                                        <h3>Access Log preview</h3>
+                                        <div class='code'>
+                                            <?php $logreader->print_lines( 'access', 10 ); ?>
+                                        </div>
+                                    </div>
+                                    <div class='column'>
+                                        <h3>Error log preview</h3>
+                                        <div class='code'>
+                                            <?php $logreader->print_lines( 'error', 10 ); ?>
+                                        </div>
+                                    </div>
+                                </div>
+                            <?php
+                            } else {
+                                echo "<p>Only users with the Super Admin role are allowed to read server logs</p>";
+                            }
+                            ?>
                         </div>
                     </div>
                     <!-- /Read server logs -->
+                    <!-- Statistics -->
+                    <div class="postbox">
+                        <h2 class="hndle">Statistics</h2>
+                        <div class="inside">
+                            <div class="row">
+                                <?php
+                                $errors = $logreader->find_match_in_log( 'access', '/HTTP\/(?:1\.1|2)\"\s+(50[24])/i');
+                                $errors_502 = 0;
+                                $errors_504 = 0;
+                                foreach ($errors as $error) {
```

Comment:
Doing business logic in the view isn't really great, the view should only print the values received.

Also, take a look at [`array_count_values`](https://www.php.net/manual/en/function.array-count-values), it should do exactly why you try to do here, but easier.